### DISCUSSION TITLE
What is the first service that we should be working on for validation testing?

### DISCUSSION THREAD

**Discussion by mcleo-d** [finos/common-cloud-controls#21] [General]
## What is the first service that we should be working on for validation testing?

This question has been posed to the Taxonomy working group by @eddie-knight from SonaType and FINOS Compliant Financial Infrastructure maintainer following #8.

Please find @eddie-knight's original question here - https://github.com/finos/common-cloud-controls/issues/15#issuecomment-1710488405

cc @mark-rushing

**Comment by eddie-knight** [comment]
Per our email thread on this topic, the CFI Reproducible Infrastructure WG has prepared a AWS-RDS ansible playbook that will be used by the Runtime Validation WG to create a taxonomy validation pack. 

If the taxonomy (even a POC taxonomy) is ready by the beginning of October, we anticipate that the RVWG should be able to have a demo validation ready in time for the Open Source in Finance Forum.

**Comment by mcleo-d** [comment]
@simonzhangbmo, @nas-hub, @rajkrishnamurthy and @eddie-knight

Here is the PDF as discussed on our last planning call.

James.

[FINOS Common Cloud Controls - OSFF 2023 Maintainer Breakout WIP](https://github.com/finos/common-cloud-controls/files/13169238/FINOS.Common.Cloud.Controls.-.OSFF.2023.Maintainer.Breakout.1.pdf)

**Comment by AdrianHammond** [comment]
This is great material @nas-hub @simonzhangbmo 

@eddie-knight are you able to share your slides please?

**Reply by mcleo-d** [reply]
Hi @AdrianHammond

This discussion was for the benefit of @simonzhangbmo who couldn't receive the rough PDF template due to its size. I will delete it when no longer needed.

Final presentations will be shared on the OSFF schedule closer to the event - https://events.linuxfoundation.org/open-source-finance-forum-new-york/program/schedule/

James.

**Comment by AdrianHammond** [comment]
okay - thanks

On Thu, 26 Oct 2023 at 12:38, James McLeod ***@***.***> wrote:

> Hi @AdrianHammond <https://github.com/AdrianHammond>
>
> This discussion was for the benefit of @simonzhangbmo
> <https://github.com/simonzhangbmo> who couldn't receive the rough PDF
> template due to its size. I will delete it when no longer needed.
>
> Final presentations will be shared on the OSFF schedule closer to the
> event -
> https://events.linuxfoundation.org/open-source-finance-forum-new-york/program/schedule/
>
> James.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/finos/common-cloud-controls/discussions/21#discussioncomment-7391778>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADA72HZTOFUJFUN6DVAMCBDYBJDTLAVCNFSM6AAAAAA4QE7C72VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TGOJRG43TQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> com>
>


-- 

Adrian Hammond, FBCS, CITP

Chief Architect

CTO Organisation

Red Hat <https://www.redhat.com/>

***@***.***
M: 07342072031

IM preference: Slack

**Comment by simonzhangbmo** [comment]
Thank you very much for the help.
I managed to get to Google Docs from my personal computer.

Regards,
Simon Zhang, Cloud & Platforms
Enterprise Architecture, BMO Financial Group
